doc/vhpidirect/examples/quickstart.rst

```diff
@@ -50,4 +50,4 @@ Since either C sources or pre-compiled ``.o`` objects can be added, in C/C++ pro
 
 If the auxillary VHPIDIRECT subprograms need to be accessed in more than one entity, it is possible to package the subprograms. This also makes it very easy to reuse the VHPIDIRECT declarations in different projects.
 
-In this example 2 different entities use a C defined ``c_printInt(val: integer)`` subprogram to print two different numbers. Subprogram declaration requirements are detailed under the :ref:`COSIM:VHPIDIRECT:Declarations` section.
\ No newline at end of file
+In this example 2 different entities use a C defined ``c_printInt(val: integer)`` subprogram to print two different numbers. Subprogram declaration requirements are detailed under the :ref:`COSIM:VHPIDIRECT:Declarations` section.
```

Comment:
I don't see what changed in this file...

Reply:
Nor do I . That is what should be there after the package merge. 🤷‍♂️

Reply:
Conflict highlighted it. 2 instead of two.

I want to add to the last sentence so that it is " Foreign subprogram declaration requirements..." I'll try to remember it in the next documentation PR